src/vcpkg/sourceparagraph.cpp

```diff
@@ -1244,7 +1244,7 @@ namespace vcpkg
                dep.constraint.type == Versions::Constraint::Type::None && !dep.host;
     }
 
-    static Json::Object serialize_manifest_impl(const SourceControlFile& scf, bool debug)
+    static Json::Object serialize_manifest_impl(const SourceControlFile& scf, bool debug, bool forExport = false)
```

Comment:
I'm really not a fan of yet another boolean here; Can the consumer just learn to handle the different outputs?

Reply:
Yeah I see your point, but the idea behind this command was to create an output that is as easy to parse as possible, and different data types for each array element does not help to make parsing easier.

Reply:
How about just change it to always be an array. @strega-nil-ms @strega-nil  Do you have strong opinions about this?

Reply:
There's no real reason to make it an array always, and it'd make a lot of descriptions add two extra unimportant lines. I'd rather the consumer learn to parse different outputs, and if not, then at least use an enum instead of a boolean parameter.